ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -164,8 +164,15 @@ wan_path_groups:
     id: 102
   - name: Equinix
     id: 103
+    dps_keepalive_interval:
+      interval: 100
+      failure_threshold: 12
   - name: Satellite
     id: 104
+    dps_keepalive_interval:
+      auto: true
+      interval: 100
+      failure_threshold: 15
```

Comment:
As @ClausHolbechArista you don't need to test that `auto` wins over `interval` in `eos_designs`, this is done already in `eos_cli_config_gen` tests - you can remove line 174 and 175 here.